pkg/controller/linstorcontrollerset/linstorcontrollerset_controller.go

```diff
@@ -338,6 +337,37 @@ func (r *ReconcileLinstorControllerSet) Reconcile(request reconcile.Request) (re
 	return reconcile.Result{RequeueAfter: time.Minute * 1}, compoundError
 }
 
+func (r *ReconcileLinstorControllerSet) reconcileResource(ctx context.Context, pcs *piraeusv1alpha1.LinstorControllerSet) error {
+	logger := logrus.WithFields(logrus.Fields{
+		"Name":      pcs.Name,
+		"Namespace": pcs.Namespace,
+		"Op":        "reconcileResource",
+	})
+	logger.Info("performing upgrades and fill defaults in resource")
```

Comment:
Maybe we should put these logs at the debug level, and just have info logging when values are actually set (inside the if) because that is the only case that is normally interesting.

Reply:
implemented on all resource upgrade/fill paths